specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/config/client.options.catalog.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: test
+bindings:
+  app0:
+    type: kafka
+    kind: client
+    options:
+      catalog:
+        test0:
+          topic: test
+          key:
+            strategy: topic
+            version: latest
+            id: 1
```

Comment:
suggest we want `strategy` + (optional) `version` OR `id` (not both), for `key` and `value`, agree?
the rationale is that `strategy` + `version` is used to resolve the `id`, or else we can specify the `id` explicitly